server/Controllers/GameControllers/snake.js

```diff
@@ -0,0 +1,16 @@
+class Snake {
+  constructor(playerId, initialPositionX, initialPositionY,
+    initialDirection, snakeColor, initialSnakeLength) {
+    this.ownerId = playerId;
+    this.headX = initialPositionX;
```

Comment:
Would it be better to use: 
`this.head = {x, y}` instead of `this.headX` and `this.headY` .. I'm not sure.

Reply:
Yes, It's better. Thanks. 👍